esmvalcore/config/extra_facets/access-mappings.yml

```diff
@@ -0,0 +1,70 @@
+# Extra facets for native ACCESS model output
+
+# A complete list of supported keys is given in the documentation (see
+# ESMValCore/doc/quickstart/find_data.rst).
+---
+
+ACCESS_ESM:
+
+  '*':
+   
+    tas:
+      raw_name: fld_s03i236
+      modeling_realm: atm
+      
+    pr:
+      raw_name: fld_s05i216
+      modeling_realm: atm
+    
+    ps:
+      raw_name: fld_s00i409
+      modeling_realm: atm
+    
+    clt:
+      raw_name: fld_s02i204
+      modeling_realm: atm
+    
+    psl:
+      raw_name: fld_s16i222
+      modeling_realm: atm
+
+    hus:
+      raw_name: fld_s30i205
+      modeling_realm: atm
+    
+    zg:
+      raw_name: fld_s30i207
+      modeling_realm: atm
+
+    va:
+      raw_name: fld_s30i202
+      modeling_realm: atm
+    
+    ua:
+      raw_name: fld_s30i201
+      modeling_realm: atm
+    
+    ta:
+      raw_name: fld_s30i204
+      modeling_realm: atm
+    
+    rlus:
+      raw_name: 
+        - fld_s02i207 
+        - fld_s02i201 
+        - fld_s03i332 
+        - fld_s02i205
+      modeling_realm: atm
+      calculate: var[0]-var[1]+var[2]-var[3]
```

Comment:
I recommend using a more readable format here, e.g.
``fld_s02i207 - fld_s02i201 + fld_s03i332 - fld_s02i205``. Instead of using ``eval`` (which leaves a lot of room for user mistakes and potential security issues), you could just write a regular expression to parses the string to get out the variables and operators, which you then apply.

Reply:
Yes, fully agree with Bouwe here. Only allow for string `raw_name` options and avoid `eval`